src/bundler.js

```diff
@@ -336,11 +343,17 @@ function processJsBundle(options, jsBundle, bundleDir, jsFiles, bundleName, cb)
                         bundleDir: bundleDir,
                         bundleStatsCollector: bundleStatsCollector,
                         sourceMap: options.sourcemaps,
+                        require: options.require,
                         siteRoot: options.siterootdirectory,
                         useTemplateDirs: options.usetemplatedirs
                     };
 
-                    if (isMustache) {
+                    if (isJson) {

```

Comment:
When we encounter `json` files, we don't want to actually add their contents to the bundle, we just want to parse them for any exports.